src/main/java/org/mongen/core/controller/CollaboratorController.java

```diff
@@ -23,40 +25,47 @@
 public class CollaboratorController {
 
 	@Autowired
-	CollaboratorService colaboratorServ;
+	CollaboratorService collaboratorServ;
 	
 	@ApiOperation(value = "Get All the Collaborators")
 	@GetMapping("/collaborators")
 	public ResponseEntity<List<Collaborator>> getCollaborators(){
-		List<Collaborator> cola = colaboratorServ.getCollaborators();
+		List<Collaborator> cola = collaboratorServ.getCollaborators();
+		return ResponseEntity.status(HttpStatus.OK).body(cola);
+	}
+
+	@ApiOperation(value = "Get All the Collaborator Types")
+	@GetMapping("/collaborators/type")
+	public ResponseEntity<List<CollaboratorType>> getCollaboratorTypes(){
+		List<CollaboratorType> cola = collaboratorServ.getCollaboratorTypes();
```

Comment:
coka?

Reply:
hahahahah we would have to set some rules for this type of variables I think, so it's more descriptive :)